Add issue template for communication (#4761)
kubernetes/community: .github/ISSUE_TEMPLATE/marketing-request.md

---

> name: Marketing Request

Author: ```suggestion
name: Contributor Comms Request
```

> about: Get help getting a message out to our community through coordinated outreach no email, social media, and other public channels.

Reviewer: ```suggestion
about: Get help getting a message out to our community through coordinated outreach on email, social media, and other public channels.
```
no -> on 👍

title: 'REQUEST: New communication about <topic>'
labels: area/contributor-comms, sig/contributor-experience

> assignees: '@jonasrosland'

Reviewer: hm -- it may not be best to assign to one person for bus-factor. Maybe ping a github team instead? would need to create one in k/org, but everyone would be notified of the request and folks could self-assign.

Author: Sounds good - did we land on a team?

---
<!--

> ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is [outlined here][multichannel].

Reviewer: ```suggestion
ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is outlined here: https://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier
```
It might be best to just link to the doc directly here, as its a comment it won't appear either when creating the issue off of it or in the issue after creation.

Filling out this form to the best of your ability will speed up the team's ability to get the word out! 
-->

**What do you want send out?** 
<!-- Please include a link whenever possible, logos, artwork, pics, diagrams, or any other assets with public links. -->

**What’s the intended outcome?**
<!-- Are you looking for people to get involved? Fill out a survey? See something of interest to them? Let us know why you care to share. -->

**What is the sponsoring SIG or community group?**

> <!-- -->

Reviewer: ```suggestion
<!-- What SIG or group is this message on behalf of? -->
```
Maybe something like that in there?

**Is this time-sensitive?** 
<!-- You can delete or skip this if it's not relevant -->

- [ ] Yes, it is **due by** <$DATE> <$TIME>
- [ ] Yes, it **must be sent** <$DATE> <$TIME>
- [ ] No, but I'd prefer <$DATE> <$TIME>

**Where do you want it to be visible?**

> <!-- Note that email via our mailinst list is the default choice for all messages. See our [multichannel communication strategy][multichannel] for more details. Note that blogs and social media must meet our guidelines available [here](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose). -->

Reviewer: ```suggestion
<!-- Note that email via our mailing list is the default choice for all messages. See our [multichannel communication strategy][multichannel] for more details. Note that blogs and social media must meet our guidelines available [here](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose). -->
```
Like the earlier link this one won't appear as a markdown link and needs to be text =/

Maybe something like:
```
Note that email via our mailing list is the default choice for all messages. See our
multichannel communication strategy for more details. Note that blogs and social
media must meet our guidelines.

Multichannel communication strategy: http://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier

Blog and Social Media Guidelines: https://git.k8s.io/community/communication/marketing-team#purpose
```

Author: Great point!

- [ ] Everywhere 🌍
- [ ] Email (kubernetes-dev@googlegroups.com)
- [ ] Blog post (with help from a [Storyteller][team])
- [ ] Twitter ([@k8scontributors](https://twitter.com/k8scontributors)
- [ ] YouTube ([Kubernetes](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg))

> - [ ] Slack (assumes `#announcements` unless otherwise mentioned)

Reviewer 2: ```suggestion
- [ ] Slack (assumes `#kubernetes-dev` unless otherwise mentioned)
```

---
name: Contributor Comms Request

> about: Get help getting a message out to our community through coordinated outreach on email, social media, and other public channels.

Author: ```suggestion
about: Get support on sending a message out to our community through coordinated outreach on email, Slack, social media, and other public channels.
```

Author: Removing redundancy.

title: 'REQUEST: New communication about <topic>'
labels: area/contributor-comms, sig/contributor-experience
assignees: 
---
<!--
ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is outlined here: https://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier

Filling out this form to the best of your ability will speed up the team's ability to get the word out! 
-->

**What do you want send out?** 

> <!-- Please include a link whenever possible, logos, artwork, pics, diagrams, or any other assets with public links. -->

Author: ```suggestion
<!-- Include logos, artwork, pics, diagrams, or any other assets with public links whenever possible. -->
```

**What’s the intended outcome?**
<!-- Are you looking for people to get involved? Fill out a survey? See something of interest to them? Let us know why you care to share. -->

**What is the sponsoring SIG or community group?**
<!-- What SIG or group is this message on behalf of? What is the best way to contact them? -->

**Is this time-sensitive?** 

> <!-- You can delete or skip this if it's not relevant -->

Author: ```suggestion
<!-- You can delete or skip this if it's not relevant. -->
```

Consistent punctuation.

**Where do you want it to be visible?**
<!-- Note that email via our mailing list is the default choice for all messages. See our
multichannel communication strategy for more details. Note that blogs and social
media must meet our guidelines.

Multichannel communication strategy: http://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier

Blog and Social Media Guidelines: https://git.k8s.io/community/communication/marketing-team#purpose -->

- [ ] Everywhere 🌍
- [ ] Email (kubernetes-dev@googlegroups.com)
- [ ] Blog post (with help from a [Storyteller](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose))
- [ ] Twitter (via [@k8scontributors](https://twitter.com/k8scontributors))
- [ ] YouTube ([Kubernetes](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg))
- [ ] Slack (assumes `#kubernetes-dev` unless otherwise mentioned)
- [ ] Contributor Calendar event entry (available [here](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America%2FLos_Angeles))
- [ ] Other (fill this out)

**Where is your first draft available?**

> <!-- Please provide a first draft of your message for all channels requested. Take your best shot and we are here to support you through getting it ready for publication (it's appreciated). 

Author: ```suggestion
<!-- Please provide a first draft of your message for all channels requested. Take your best shot and we are here to support you through getting it ready for publication (it's appreciated). Links to Google Docs or Hackmd.io pages are best for easy editing.
```

Also, include any social media tags of relevant people (e.g. Twitter handles of people you want to give a shout out on Twitter)-->

**Lastly, who's the best point of contact?** 
<!-- If you are the point of contact, please confirm it's you. Please include both GitHub username and Slack username. -->

> 

Author: ```suggestion
<!-- Last but not least, thank you! We're excited to help get the word out. -->
```
